src/TwigComponent/src/Twig/TwigPreLexer.php

```diff
@@ -259,6 +263,10 @@ private function consumeAttributes(string $componentName): string
      */
     private function consume(string $string): bool
     {
+        if ($string[0] !== $this->input[$this->position]) {
+            return false;
+        }
```

Comment:
This change alone increases the `cache:warmup` time by 11%:

<img src="https://github.com/symfony/ux/assets/330436/0a3982db-90fe-496f-879d-8a69d2609a60" width=500/>